.github/workflows/ci.yaml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version:
+          - 3.7
+          - 3.8
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # full history for setuptools_scm
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
```

Comment:
Why not `v2`?

Reply:
My bad! I was quickly copying and pasting from another project that I hadn't updated lately. Will fix up later 👍